playbooks/openshift/roles/common/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: reload firewalld
```

Comment:
Is restarting the firewall a common role handler or only particular to certain role(s)?